pkg/katautils/config_test.go

```diff
@@ -167,7 +169,7 @@ func createAllRuntimeConfigFiles(dir, hypervisor string) (config testRuntimeConf
 		GuestHookPath:         defaultGuestHookPath,
 		VhostUserStorePath:    defaultVhostUserStorePath,
 		SharedFS:              sharedFS,
-		VirtioFSDaemon:        "/path/to/virtiofsd",
+		VirtioFSDaemon:        virtiofsdaemon,
```

Comment:
@c3d, here it should be `virtioFSdaemon`, as that's the naming used above in this file.

Here's a log of travis, which didn't return back to us, pointing to this error:
https://travis-ci.org/github/kata-containers/runtime/jobs/695051557#L1070

Reply:
Retrying 😄

Reply:
Travis has passed, but results were not reported back:
https://travis-ci.org/github/kata-containers/runtime/builds/695129068